Marlin/src/lcd/e3v2/proui/proui_extui.cpp

```diff
@@ -93,7 +93,7 @@ namespace ExtUI {
   void onUserConfirmRequired(const char * const cstr) {
     // TODO: A version of this method that takes an icon and button title,
     // or implement some kind of ExtUI enum.
-    onUserConfirmRequired(ICON_Continue_1, cstr, GET_TEXT_F(MSG_USERWAIT));
+    dwinPopupContinue(ICON_Continue_1, FSTR_P(cstr), GET_TEXT_F(MSG_USERWAIT));
```

Comment:
I see many other calls to `onUserConfirmRequired`, but only one other call to `dwinPopupContinue`.

Can you explain the reason for this change, since this is one of only two places in the code which calls `dwinPopupContinue`?

Reply:
`onUserConfirmRequired` calling on itself would lead to `dwinPopupConfirm`, with a button saying "Confirm"
this change allows it to be correct with button saying "Continue"